doc/changes/latest.inc

```diff
@@ -20,7 +20,10 @@ Enhancements
 Bugs
 ~~~~
 
-- Fix bug with :func:`mne.viz.plot_source_estimates` when using the PyVista backend where singleton time points were not handled properly, by `Eric Larson`_
+- Fix bug with :func:`mne.viz.plot_source_estimates` when using the PyVista backend where singleton time points were not handled properly by `Eric Larson`_ (`#8285 <https://github.com/mne-tools/mne-python/pull/8285>`_)
+
+- :func:`mne.io.read_raw_edf` now supports EDF files with invalid recording dates by `Clemens Brunner`_ (`#8283 <https://github.com/mne-tools/mne-python/pull/8283>`_)
```

Comment:
❤️❤️❤️ @ PR numbers.

Reply:
😄  - @hoechenberger could you take care of backporting this PR after it is merged?

Reply:
Wow am I the backport guy now? 🧐 
Just kidding, will do, no Problem! 😉

Reply:
In our docs you can just do:
```
:gh:`8283`
```
and it should produce the link above because of

https://github.com/mne-tools/mne-python/blob/master/doc/sphinxext/gh_substitutions.py

Reply:
(see for example the bottom of entries in https://github.com/mne-tools/mne-python/blob/master/doc/overview/roadmap.rst#clustering-statistics-api and the resulting output https://mne.tools/dev/overview/roadmap.html

Reply:
```suggestion
- Fix bug with :func:`mne.viz.plot_source_estimates` when using the PyVista backend where singleton time points were not handled properly by `Eric Larson`_ (`#8285 <https://github.com/mne-tools/mne-python/pull/8285>`_)

- :func:`mne.io.read_raw_edf` now supports EDF files with invalid recording dates by `Clemens Brunner`_ (:gh:`8283`)
```

Reply:
> In our docs you can just do:
> 
> ```
> 
> :gh:`8283`
> 
> ```
> 
> and it should produce the link above because of
> 
> 
> 
> https://github.com/mne-tools/mne-python/blob/master/doc/sphinxext/gh_substitutions.py

Mind = blown!

🤯

@sappelhoff you seeing this stuff???? ****WANT****

Reply:
Nice, thanks @larsoner for pointing out this gem! And thanks @hoechenberger for backporting, I do like the title "backporting guy" though 😆 !